pipeline-runner/data-raw/sysdata.R

```diff
@@ -0,0 +1,9 @@
+# constrants used in GEM2S
+gem2s <- list(
+    max.edrops.fdr = 0.001,
+    max.empty.counts = 100,
+    max.empty.drops = 50
+)
+
+
+usethis::use_data(gem2s, internal = TRUE)
```

Comment:
you can also make a similar file for that qc config that currently is added in step 6 of the gem2s pipeline.

Reply:
step 6 of gem2s is creating qc config that is uploaded to AWS DynamoDB. This is a set of constants that are used by gem2s. I agree that we should do the same for QC constants although I think that could be part of a larger refactor that we want to undertake for QC pipeline as well and is probably best done as a seperate ticket to keep the scope of this ticket on GEM2S

Reply:
sure, happy to do it as part of another ticket, this PR is definitely getting way too big